src/Tickets/Emails/Admin/Emails_Tab.php

```diff
@@ -172,4 +224,126 @@ public function get_fields(): array {
 		 */
 		return apply_filters( 'tec_tickets_emails_settings_fields', $fields );
 	}
+
+	/**
+	 * Check if currently editing email.
+	 * 
+	 * @since TBD
+	 *
+	 * @param Email_Abstract $email
+	 * 
+	 * @return boolean
+	 */
+	public function is_editing_email( $email = null ) {
+		if ( ! $this->is_on_section( static::$edit_section_slug ) ) {
+			return false;
+		}
+
+		// Get `tc-edit` query string from URL.
+		$editing_email  = tribe_get_request_var( static::$key_edit_id );
```

Comment:
We can possibly simplify all this by using just `section` as a parameter for the different emails we have.

Reply:
In the future, are we certain that `section` will only be used for editing emails?

Reply:
I'll assume that we will not be using `section` for anything else other than editing emails. I was just trying to keep things flexible for the future.